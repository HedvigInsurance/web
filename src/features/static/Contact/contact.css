.Contact {
  height: 100%;
  width: 100%;
}

.Contact__header {
  font-size: 28px;
  line-height: 32px;
  font-family: "Merriweather";
  color: #0f007a;
  font-weight: 400;
  margin: 2em 0;
  text-align: center;
}

.Contact__article {
  justify-content: center;
  padding: 0 0 2em;
}

.Contact__text {
  width: 100%;
  text-align: center;
}

@media screen and (min-width: 1024px) {
  .Contact__article {
    padding: 5em 0;
  }
}
