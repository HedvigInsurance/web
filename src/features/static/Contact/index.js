import React from "react"
import { Helmet } from "react-helmet"

import Header from "../../../components/Header";
import Footer from "../../../components/Footer"
import Hero from "../../../components/Hero"
import { SayHi } from "../common"
import "./contact.css"

export default () => (
  <main className="Contact">
    <Helmet>
      <title>Kontakt | Hedvig</title>
    </Helmet>
    <Header headerRight={<SayHi/>}/>
    <Hero
      imageUrl="/assets/web/Images/map.png"
      alt="Map centered on Hedvigs Office at Norrsken"
    />
    <article className="pure-g Contact__article">
      <h1 className="pure-u-1-1 Contact__header">Vill du komma i kontakt med oss?</h1>
      <section className="pure-u-1-1 Contact__text">
        <a href="mailto:hedvig@hedvig.com">hedvig@hedvig.com</a><br/>
        <a href="mailto:press@hedvig.com">press@hedvig.com</a><br/>
        <a href="mailto:careers@hedvig.com">careers@hedvig.com</a><br/>
      </section>
    </article>
    <Footer />
  </main>
)
