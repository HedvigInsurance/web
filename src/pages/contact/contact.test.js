import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Contact from './index';

describe('<Contact />', () => {
  it('Should render correctly', () => {
    const component = shallow(<Contact />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
