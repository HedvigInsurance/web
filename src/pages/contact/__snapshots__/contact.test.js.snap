// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Contact /> Should render correctly 1`] = `
<main
  className="Contact"
>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Kontakt | Hedvig
    </title>
  </HelmetWrapper>
  <Header />
  <article
    className="pure-g Contact__article"
  >
    <h1
      className="pure-u-1-1 Contact__header"
    >
      Vill du komma i kontakt med oss?
    </h1>
    <section
      className="pure-u-1-1 Contact__text"
    >
      <a
        href="mailto:hedvig@hedvig.com"
      >
        hedvig@hedvig.com
      </a>
      <br />
      <a
        href="mailto:press@hedvig.com"
      >
        press@hedvig.com
      </a>
      <br />
      <a
        href="mailto:careers@hedvig.com"
      >
        careers@hedvig.com
      </a>
      <br />
    </section>
  </article>
  <Footer />
</main>
`;
